app/controllers/CyaCreateRecordController.scala

```diff
@@ -59,7 +59,7 @@ class CyaCreateRecordController @Inject() (
 
   // TODO redirect to correct location and submit data
   def onSubmit(): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    Redirect(routes.IndexController.onPageLoad)
+    Redirect(routes.CreateRecordSuccessController.onPageLoad().url)
```

Comment:
Can we now remove the //TODO redirect to correct location above?

Reply:
This is done :)